src/runtime_src/core/common/api/xrt_xclbin.cpp

```diff
@@ -27,7 +27,10 @@
 #include "core/common/query_requests.h"
 #include "core/include/xclbin.h"
 #include "core/common/xclbin_parser.h"
+#include "xclbin_int.h"
+#include <iostream>
 #include <fstream>
+#include <unordered_set>
 
 #ifdef _WIN32
 # include "windows/uuid.h"
```

Comment:
Remove the includes of system uuid.h and instead use 'experimental/uuid.h' which wraps uuid functionality in a xrt::uuid c++ object